src/Downloader.php

```diff
@@ -129,6 +138,12 @@ public function getRecipes(array $operations): array
 
         $data = [];
         foreach ($bodies as $body) {
+            // add default values in case the response is empty
+            $body += [
+                'manifests' => [],
+                'vulnerabilities' => [],
+                'locks' => [],
+            ];
```

Comment:
I don"t think this is needed and should probably be removed.

Reply:
L135 populates $bodies with empty arrays, that's why.

Reply:
Pfff but the "if" L135 also prevents it...

Reply:
But this code works right now without adding this, right?

Reply:
Removed